src/app/models/form.server.model.ts

```diff
@@ -4,7 +4,6 @@ import { Model, Mongoose, Schema, SchemaOptions } from 'mongoose'
 import validator from 'validator'
 
 import { FORM_DUPLICATE_KEYS } from '../../shared/constants'
-import { validateWebhookUrl } from '../../shared/util/webhook-validation'
```

Comment:
Wow have we never used it in the frontend?

Reply:
nope. weird huh.